status_test.go

```diff
@@ -1,77 +1,112 @@
 package mcpinger
 
 import (
-	"io/ioutil"
+	"os"
 	"path/filepath"
 	"testing"
 )
 
 const (
-	TestDataDir       = "testdata" // Name of testdata directory
-	VersionName       = "1.13.2"
-	Protocol          = 404
-	Description       = "Hello world"
-	MaxPlayers        = 100
-	OnlinePlayers     = 5
-	PlayerSampleCount = 1
-	PlayerSampleName  = "Raqbit"
-	PlayerSampleUuid  = "09bc745b-3679-4152-b96b-3f9c59c42059"
-	Favicon           = "data:image/png;base64,<data>"
+	TestDataDir = "testdata" // Name of testdata directory
 )
 
 func TestParseServerInfo(t *testing.T) {
-	files := []string{
-		"info.json",
-		"info_description_string.json",
+	tests := []struct {
+		File              string
+		VersionName       string
+		Protocol          int32
+		Description       string
+		MaxPlayers        int32
+		OnlinePlayers     int32
+		PlayerSampleCount int
+		PlayerSampleName  string
+		PlayerSampleUuid  string
+		Favicon           string
+	}{
```

Comment:
I hope you don't mind, but I thought I'd change this to a table driven test. If you would prefer I can revert this and add the new scenario as a separate test case.